arm-keyvault/2015-06-01/swagger/keyvault.json

```diff
@@ -0,0 +1,463 @@
+            "schema": {
+              "$ref": "#/definitions/VaultCreateOrUpdateParameters"
+            },
+            "description": "Parameters to create or update the vault"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created or updated vault",
+            "schema": {
+              "$ref": "#/definitions/Vault"
+            }
+          },
+          "200": {
+            "description": "Created or updated vault",
+            "schema": {
+              "$ref": "#/definitions/Vault"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Vaults"
+        ],
+        "operationId": "Vaults_Delete",
+        "description": "Deletes the specified Azure key vault.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Resource Group to which the vault belongs."
+          },
+          {
+            "name": "vaultName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vault to delete"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted vault",
+            "schema": {
+              "$ref": "#/definitions/Vault"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Vaults"
+        ],
+        "operationId": "Vaults_Get",
+        "description": "Gets the specified Azure key vault.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Resource Group to which the vault belongs."
+          },
+          {
+            "name": "vaultName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vault."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Retrieved vault",
+            "schema": {
+              "$ref": "#/definitions/Vault"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/resources": {
+      "get": {
+        "tags": [
+          "Vaults"
+        ],
+        "operationId": "Vaults_List",
```

Comment:
I understand that it is breaking change. IMHO, it would be ok to break them one time right now while moving from Hyak to AutoRest rather than breaking them again.